src/cargo_loading_service.cpp

```diff
@@ -74,17 +74,23 @@ void CargoLoadingService::execCargoLoading(
   infra_id_ = request->id;
   service_result_ = ExecuteInParkingTask::Response::SUCCESS;
```

Comment:
```suggestion
  if (request->id < static_cast<uint8_t>(InfraIdLimit::MIN) ||
      request->id > static_cast<uint8_t>(InfraIdLimit::MAX)) {
      RCLCPP_WARN(this->get_logger(), "Invalid ID = %d", request->id);
      response->state = ExecuteInParkingTask::Response::FAIL;
      return;
  }
  infra_id_ = request->id;
  service_result_ = ExecuteInParkingTask::Response::SUCCESS;
```
- service call時のINVALID IDは、サービスがまだ開始されない段階であるため、service_resultの失敗による設備への終了指示（0b00）を送る必要がないです。responseのみFailで返してください。
- ソースコードの見通しを良くするため、early returnを心がけてください。
- INVALID ID時はサービス開始すらしていないのでinfra_id_の変更も不要です。